exercises/concept/valentines-day/.meta/exemplar/src/ValentinesDay.hs

```diff
@@ -0,0 +1,36 @@
+module ValentinesDay (rateActivity, Approval (..), Cuisine (..), Genre (..), Activity (..)) where
+
+data Approval
+  = Yes
+  | No
+  | Maybe
+  deriving (Show, Eq)
+
+data Cuisine
+  = Korean
+  | Turkish
+
+data Genre
+  = Crime
+  | Horror
+  | Romance
+  | Thriller
+
+data Activity
+  = BoardGame
+  | Chill
+  | Movie Genre
+  | Restaurant Cuisine
+  | Walk Int
+
+rateActivity :: Activity -> Approval
+rateActivity activity =
+  case activity of
```

Comment:
This implementation uses `case`, which I don't think is being taught in the prerequisites, correct? If so, we'll need to add another exercise to teach that or rewrite this to use the pattern matching functionality that was already introduced.

Reply:
`case` is introduced in this concept (last paragraph of introduction.md)